engine/include/cubos/engine/prelude.hpp

```diff
@@ -7,6 +7,7 @@
 #include <cubos/core/ecs/cubos.hpp>
 #include <cubos/core/ecs/entity/entity.hpp>
 #include <cubos/core/ecs/system/system.hpp>
+#include <cubos/core/memory/opt.hpp>
```

Comment:
```suggestion
```
This is already being included by system.hpp, as it is one of its possible arguments. No need to add it here.

Reply:
I understand why, at the same time, It makes me a bit tedious sometimes trying to find where stuff comes from, when the file doesn't include what it needs. Technically, its included once because of `pragma once` or whatever guard we are using. But ok, keeping it consistent with the other files!